src/main/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayout.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.hmcts.reform.logging.layout;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.ThrowableProxy;
+import ch.qos.logback.core.CoreConstants;
+import ch.qos.logback.core.LayoutBase;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.logging.exception.AbstractLoggingException;
+import uk.gov.hmcts.reform.logging.exception.InvalidExceptionImplementation;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.NoSuchElementException;
+
+public class ReformLoggingLayout extends LayoutBase<ILoggingEvent> {
+
+    private Logger log = LoggerFactory.getLogger(ReformLoggingLayout.class);
+
+    private Calendar calendar = Calendar.getInstance();
+
+    private SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
+
+    /**
+     * By default don't require thread to be logged.
+     */
+    private boolean requireThread = false;
+
+    /**
+     * By default require an AlertLevel tag to be present for any exception log.
```

Comment:
commented on the wrong line before I meant this one.
i.e. using this would require all teams to update their code?

Reply:
it will still log the message/exception but in case an error is logged with exception - additionally `InvalidExceptionImplementation` is logged in. transition should be easy